### DISCUSSION TITLE
video tutorial.

### DISCUSSION THREAD

**Discussion by hectorzin** [elchininet/custom-sidebar#157] [Show and tell]
Hi Iván, I’ve made a short tutorial video about this interesting integration for Home Assistant. It’s not intended as spam, just informative, in case you find it appropriate to possibly include it in the documentation. Feel free to delete this message if you find it bothersome or spammy. I hope the video helps more people become aware of the integration you've created.

The video: https://www.youtube.com/watch?v=oecwBjPmlYo

Best regards!

**Comment by elchininet** [comment]
Hi @hectorzin,

I don't find it bothersome or spammy. It is something good for the community, so more people can be aware of your channel and learn more about home automation and at the same time they can discover new Home Assiatant plugins/integrations through your videos.

I'll add it to the documentation to a section dedicated to this when I have an opportunity (stating clearly that it is in our mother tongue to avoid confussions) 🙂

Un saludo 👍
